index.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module 'webext-options-sync' {
+	interface DefineOptions {
+		defaults: {
+			disabledFeatures: string;
+			customCSS: string;
+			personalToken: string;
+			logging: boolean;
+		};
+		migrations: [(options: {disabledFeatures: string}) => void, () => void];
+	}
+
+	export default class OptionsSync {
+		static migrations: {
+			removeUnused: () => void;
+		}
+
+		getAll: <T>() => T;
+
+		setAll: (newOptions: unknown) => Promise<void>;
```

Comment:
I don't know what should be in the newOptions for both these functions. Is the type `DefineOptions`?

Reply:
> Type: object
> A map of default options as strings or booleans.